docs/docs/contribute/code/03.building-hasten.md

```diff
@@ -0,0 +1,11 @@
+---
+authors: Fragcolor & contributors
+license: CC-BY-SA-4.0
+---
+
+# 3. Building Hasten
+
+// TODO //
```

Comment:
Let's hide those unfinished pages. One way (aside from removing them) would be to rename them to a different extensions than `.md`, for instance `.md.tmp` or `.md.todo`.

Reply:
Yes, that's a good idea. Renamed these files with md.todo extension (and will follow this convention in the future).